src/types/json.h

```diff
@@ -217,6 +217,42 @@ struct JsonValue {
     return results;
   }
 
+  StatusOr<std::vector<std::string>> GetBytes(std::string_view path, JsonStorageFormat format,
```

Comment:
Why `vector<string>` instead of `vector<size_t>`?

Reply:
Because currently ArrayOfBulkStrings only supports string types, if modified to a template, it cannot meet the writing requirements of std:: initializer_list. Do you suggest that I add another method, std: vector<size_t>?

Reply:
You can use a loop to dump RESP output. It doesn't matter whether you use ArrayOfBulkStrings or not.

Reply:
ok thanks